elasticsearch/es_client_v7.go

```diff
@@ -301,3 +302,83 @@ func (es *ESClientV7) GetTotalDiskUsage(ctx context.Context) (string, error) {
 
 	return totalDiskUsage, nil
 }
+
+func (es *ESClientV7) GetDBUserRole(ctx context.Context) (error, bool) {
+	req := esapi.SecurityGetRoleRequest{
+		Name: []string{"custom-user"},
```

Comment:
Use constants instead of hard-coded strings. You can add them here - https://github.com/kubedb/db-client-go/blob/f83070a8facbf2299cdfda3064874b4e8e37c44b/elasticsearch/api.go#L27C6-L27C6